src/hub.js

```diff
@@ -195,10 +195,16 @@ const onReady = async () => {
     if (NAF.connection.adapter && NAF.connection.adapter.localMediaStream) {
       NAF.connection.adapter.localMediaStream.getTracks().forEach(t => t.stop());
     }
-    hubChannel.disconnect();
+    if (hubChannel) {
+      hubChannel.disconnect();
+    }
     const scene = document.querySelector("a-scene");
-    scene.renderer.animate(null); // Stop animation loop, TODO A-Frame should do this
-    document.body.removeChild(scene);
+    if (scene) {
```

Comment:
What s the case this and the next check are guarding against? Why would the scene not exist?

Reply:
oh, that's why? I hadn't considered that. makes (the most) sense. (should a comment be added?)